### PR TITLE
orchestrator: Fix spelling

### PR DESCRIPTION
**Pull request by jsoref**
This is based on https://github.com/ceph/ceph/pull/46824?notification_referrer_id=NT_kwDOACBWLLIzODY3MDU2MDc2OjIxMTkyMTI#issuecomment-1523060850 and https://github.com/ceph/ceph/pull/46824?notification_referrer_id=NT_kwDOACBWLLIzODY3MDU2MDc2OjIxMTkyMTI#issuecomment-1523142686

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket -- I'm not sure.
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

I can provide a test for this, but I'd rather not do it initially. As it should involve the project discussing it.

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by jsoref** [comment]
Note: I'm more than happy to squash, but I'd like to get buy-in for the changes first. Generally for changes w/ >10 things there's at least one that someone finds objectionable. And it's much easier for me to drop / fix them before I squash than after (and similarly, if I need to rebase, which I did for the parent PR, it's considerably more error prone if it's already squashed).

**Comment by jsoref** [comment]
Hmm, it looks like my attempt to follow codeowners didn't work properly... I'll see which other things should be dropped...

File: qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml
Comment (jsoref) [inline qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml:62]: this is duplicate word detection. It's possible the right correction should be:
```diff
-new new
+a new
```

File: qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml
Comment (jsoref) [inline qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml:69]: There was a request to expand contractions. I only did it in one case. I can do it more generally, but given how large the main PR is, and that even when split, the PRs are still sizable, my preference is to not add too many changes at once. I can always return (especially once people are more comfortable w/ me making large changes).

File: src/cephadm/cephadm.py
Comment (adk3798) [inline src/cephadm/cephadm.py:5021]: This fixed the spelling here, but we also need to fix it in the corresponding unit test to get the test passing (that's the reason make check is failing). Here's a sample commit changing the test to get it passing https://github.com/adk3798/ceph/commit/abd9b27454c6b12eb131384379adee710504396e

**Review by adk3798 — COMMENTED** [review]
Mostly LGTM outside of the failing unit test. Can do a proper qa run once the other couple comments are addressed

File: qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml
Comment (jsoref) [reply qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml:69]: I initially checked two letter words, but it turned out that the false positive rate was sufficiently poor that it just wasn't a good use of my time, so I changed the default word threshold to 3 characters. Happy to fix things people spot.

Comment (adk3798) [reply qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml]: Okay, makes sense. I only even saw this one since it was fixing another word in the line. Thanks for fixing it up.

File: src/cephadm/cephadm.py
Comment (jsoref) [reply src/cephadm/cephadm.py:5021]: So, the reason it wasn't changed is that the file ended up in excludes:
https://github.com/ceph/ceph/blob/32fcb4a03159d25547dd7f87e5d7176e1a0e5ed3/.github/actions/spelling/excludes.txt#L155

I'm not sure offhand _why_ it ended up there. I could review the runs in https://github.com/check-spelling/ceph/actions/workflows/spelling.yml to try to figure out why. But, I'd rather not spend the time doing that unless we think it indicates a bug in the analysis/detection of things to skip, and that should turn up pretty quickly when I remove the excludes entry.

**Review by adk3798 — APPROVED** [review]

**Comment by adk3798** [comment]
https://pulpito.ceph.com/adking-2023-05-03_12:22:42-orch:cephadm-wip-adk-testing-2023-05-02-1906-distro-default-smithi/

5 failures:

- 2 failures on staggered upgrade test. Combination of https://tracker.ceph.com/issues/58535 plus the attempted fix not quite working yet
- 1 failure in the test_cephadm task. Related to a PR in the run changing the `cephadm version` command
- 1 failure deploying jaegar-tracing. Still need to investigate this, but it's not new to this run, so shouldn't block merging.
- 1 failure making an OSD in the mds upgrade sequence `ERROR: error creating empty object store in /var/lib/ceph/osd/ceph-5/: (5) Input/output error`. These pop up occasionally in hte lab, most likely as a result of us using split nvme devices for OSDs. Nothing that would block merging. 

Overall, failures should only block merging for the 2 PRs related to the staggered upgrade and cephadm version changes. Everything else should be fine.

**Pull request merged**
